content/60.methods.md

```diff
@@ -1,50 +1,101 @@
+
+Given the limitations imposed upon scientists by the COVID-19 pandemic and social distancing measures that had most scientists (including students) working from home for much of 2020, community building across disciplines and across career stages was a priority of the project.
 Contributors were recruited by word of mouth and on Twitter.
 Existing efforts to train early-career scientists were also integrated: Appendix A contains summaries written by the students, post-docs, and faculty of the Immunology Institute at the Mount Sinai School of Medicine [@url:https://github.com/ismms-himc/covid-19_sinai_reviews; @doi:10.1038/s41577-020-0319-0], and two of the authors were recruited through the American Physician Scientist Association's Virtual Summer Research Program [@url:https://www.physicianscientists.org/page/summer-research-pilot-program].
-The project was managed through GitHub [@url:https://github.com/greenelab/covid19-review] using Manubot [@doi:10.1371/journal.pcbi.1007128] to continuously generate a version of the manuscript online [@url:https://greenelab.github.io/covid19-review].
-Contributors developed text that was proposed through GitHub's pull request system and then reviewed and approved by at least one other author.
-While this document reflects the current version of record, the online version will continue to be developed as information about the pandemic emerges.
-Below, we will describe the processes used to synthesize the literature.
 
-### Technical Infrastructure
+Interested parties were encouraged to contribute in a number of ways.
+One option was to submit articles of interest as issues in the GitHub repository.
+Articles were classified as _diagnostic_, _therapeutic_, or _other_, and a template was developed to guide the review of papers and preprints in each category.
+Following a framework often used for assessing medical literature, the review consisted of examining methods used in each relevant article, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates [@doi:10.5014/ajot.60.4.367].
+For examples of each template, please see Appendices B-D.
+Another option was to contribute or edit text using GitHub's pull request system.
+Each pull request was reviewed and approved by at least one other author.
+Manubot also provides a functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
+The GitHub implementation allowed users to contribute either using git on the command line or using the GitHub user interface, and we developed documentation for users with less experience with this platform.
 
-#### Collaborative Writing and Manuscript Generation
+Few researchers in biological and medical fields are trained in version control tools such as git <!--To Do: Find some sort of reference for this, software carpentry?-->
```

Comment:
A nice article but it isn't supporting what you say here https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004668

Maybe something [citing it](https://scholar.google.com/scholar?cites=11154469807022103800&as_sdt=5,50&sciodt=0,50&hl=en) would be more appropriate?